packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -202,21 +233,28 @@
 
 :host([selection-appearance="border"][layout="horizontal"]) {
   .container.selected {
-    box-shadow: inset 0 -4px 0 0 var(--calcite-color-brand);
+    --calcite-internal-tile-shadow: inset 0 -4px 0 0 var(--calcite-color-brand);
   }
 }
 
 :host([selection-appearance="border"][layout="vertical"]) {
   .container.selected {
-    box-shadow: inset 4px 0 0 0 var(--calcite-color-brand);
+    --calcite-internal-tile-shadow: inset 4px 0 0 0 var(--calcite-color-brand);
+  }
+}
+
+:host([href]:active:not([disabled])) {
+  .container {
+    box-shadow: 0 0 0 3px var(--calcite-tile-border-color);
   }
 }
 
 :host([href]:focus:not([disabled])),
 :host([href]:hover:not([disabled])) {
-  --calcite-tile-border-color: var(--calcite-color-text-link);
-  --calcite-tile-heading-text-color: var(--calcite-color-text-link);
-  --calcite-icon-color: var(--calcite-color-text-link);
+  --calcite-internal-tile-border-color: var(--calcite-color-text-link);
+  --calcite-internal-tile-heading-text-color: var(--calcite-color-text-link);
+  --calcite-internal-tile-icon-color: var(--calcite-color-text-link);
+  --calcite-ui-icon-color: var(--calcite-color-text-link);
```

Comment:
`ui-icon-color` is deprecated

Reply:
Not sure if removing this would constitute a breaking change or not, so I hesitate to remove that.  All of these changes straight-up copied over from the tokens branch, so my guess is this is here for a reason to prevent a breakage.  Unless we're ok with breaking changes on these PRs, I suppose it would be fine to remove it.  @jcfranco thoughts?

Reply:
the `--calcite-ui-icon-color` is deprecated but still available on the calcite-icon however it is overwritten by `--calcite-icon-color`. However we don't just want to replace `--calcite-ui-icon-color` with `--calcite-icon-color` in this case because that's not how we're styling calcite-icon as a sub-component in other components.

Instead this should be something like the following.

```css
calcite-icon {
  color: var(--calcite-tile-icon-color, var(--calcite-color-text-link));
}
```